luigi/hadoop.py

```diff
@@ -689,6 +704,17 @@ def jobconfs(self):
             jcs.append('mapred.reduce.tasks=%s' % self.n_reduce_tasks)
         return jcs
 
+    @cached_property
+    def serialize(self):
+        return getattr(
+            DataInterchange, self.data_interchange_format)['serialize']
```

Comment:
I sense that you shouldn't use getattr in this case. Can't DataInterChange just be a dictionary? Or can't you change hte type of `data_interchange_format` to be the dict itself?